libs/httpext/src/lib/http-converter.spec.ts

```diff
@@ -0,0 +1,88 @@
+import { HttpHeaders, HttpRequest, HttpParams } from '@angular/common/http';
+
+import { Request } from './http';
+import { fromNgReq, toNgReq } from './http-converter';
+
+describe('fromNgReq', () => {
+  it('should convert HttpRequest to Request object', () => {
+    const requests = [
+      new HttpRequest(
+        'POST',
+        'https://angular.io',
+        { data: 'hello world' },
+        { headers: new HttpHeaders({ Authorization: 'token' }) }
+      ),
+      new HttpRequest('GET', 'https://wikipedia.com', {
+        params: new HttpParams().set('id', '1')
+      })
+    ];
+
+    const expectations: Request<unknown>[] = [
+      {
+        url: 'https://angular.io',
+        method: 'POST',
+        body: { data: 'hello world' },
+        headers: { Authorization: 'token' },
+        params: {}
+      },
+      {
+        url: 'https://wikipedia.com',
+        method: 'GET',
+        body: null,
+        headers: {},
+        params: { id: '1' }
+      }
+    ];
+
+    requests.forEach((ngRequest, i) =>
+      expect(fromNgReq(ngRequest)).toEqual(expectations[i])
+    );
```

Comment:
Great job!
I prefer avoiding for loops and stuff like for tests in general. What do you think?
When the tests fail, we'll end up with an error at line 38 and we'll have to "investigate".

Maybe it's two different tests by the way "should test post..." "should test get..." etc...

Reply:
✅specs are now split
let me know if it's ok for you